dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -114,13 +114,15 @@ class TypeOfChange(Enum):
 
 
 # defines the precedence order for provider version bumps
-# BREAKING_CHANGE > FEATURE > BUGFIX > MIN_AIRFLOW_VERSION_BUMP > MISC > DOCUMENTATION > SKIP
+# BREAKING_CHANGE > FEATURE > MIN_AIRFLOW_VERSION_BUMP > BUGFIX > MISC > DOCUMENTATION > SKIP
```

Comment:
from the which type of release to make, it's actually
`BREAKING_CHANGE > FEATURE >= MIN_AIRFLOW_VERSION_BUMP > BUGFIX >= MISC > DOCUMENTATION > SKIP`
but not sure if it's further clarify

Reply:
Hmm, if it helps you, sure